roles/installer/ospd/loadbalancer/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: create VLAN interface for internal_api network
+  become_user: root
+  template: src=vlan-internal_api.cfg.j2
+            dest=/etc/sysconfig/network-scripts/ifcfg-{{ installer.overcloud.loadbalancer.network.undercloud.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.InternalApiNetworkVlanID }}
+  when: installer.overcloud.loadbalancer.network.internal_api is defined
+
+- name: Start internal_api interface
+  shell: sudo /usr/sbin/ifup {{ installer.overcloud.loadbalancer.network.undercloud.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.InternalApiNetworkVlanID }}
```

Comment:
I don't know how this works for you, but it's better to quote every string that contains var reference. So:
`shell: "sudo /usr/sbin/ifup {{ installer.overcloud.loadbalancer.network.undercloud.oc_interface }}.{{ installer.overcloud.network.template.content.parameter_defaults.InternalApiNetworkVlanID }}"`